packages/local-signing-manager/src/lib/local-signing-manager.ts

```diff
@@ -119,9 +119,9 @@ export class LocalSigningManager implements SigningManager {
   /**
    * @hidden
    */
-  private constructor(accounts: PrivateKey[]) {
+  private constructor(accounts: PrivateKey[], args: { curveType?: Curve } = {}) {
```

Comment:
This is private, so people aren't actually going to be able to use this change unless they disable TypeScript. the reason for this is to allow for async stuff to happen when making an instance.

You need to add the param into the `create` method.

Reply:
(style) For me I like this a bit more. If we are providing a default object we can use that and cut down on a code branch:

```ts
args: { curveType: Curve } = { curveType: 'ed25519' }
...
type: args.curveType
```

Reply:
Sure, thanks for the comment.